core/01_core.js

```diff
@@ -130,9 +130,15 @@
     return res;
   }
 
-  function opAsync(opName, arg1 = null, arg2 = null) {
+  function opAsync(opName, arg1 = null, arg2 = null, unref = false) {
     const promiseId = nextPromiseId++;
-    const maybeError = opcallAsync(opsCache[opName], promiseId, arg1, arg2);
+    const maybeError = opcallAsync(
+      opsCache[opName],
+      promiseId,
+      arg1,
+      arg2,
+      unref,
+    );
```

Comment:
This change was done so that ops that are unref by default don't have to call into Rust immediately after dispatch to mark them as "unrefed"

Reply:
I probably would keep the old signature. The only "unref-by-default" ops are signal listeners and they're not a performance concern.

Reply:
Reverted.